core/src/main/java/org/apache/calcite/tools/RelBuilder.java

```diff
@@ -2577,7 +2578,9 @@ public GroupKey alias(String alias) {
         String alias, ImmutableList<RexNode> operands,
         ImmutableList<RexNode> orderKeys) {
       this.aggFunction = Objects.requireNonNull(aggFunction);
-      this.distinct = distinct;
+      // When AggregateCall.distinct is false or getDistinctOptionality() is IGNORED,
+      // no distinct operation is required. In addition, a distinct operation is required.
```

Comment:
change the comment to 
`distinct value is honored only when the aggregate function's distinct optionality is not IGNORED.`
and in another place.

Reply:
> change the comment to
> `distinct value is honored only when the aggregate function's distinct optionality is not IGNORED.`
> and in another place.

hi @hsyuan Thank you very much, I've already changed it.